src/main/java/twilightforest/world/ChunkGeneratorTwilightForest.java

```diff
@@ -812,7 +812,9 @@ public void populate(int chunkX, int chunkZ) {
 		long l1 = (rand.nextLong() / 2L) * 2L + 1L;
 		long l2 = (rand.nextLong() / 2L) * 2L + 1L;
 		rand.setSeed(chunkX * l1 + chunkZ * l2 ^ world.getSeed());
-
+		
+		net.minecraftforge.event.ForgeEventFactory.onChunkPopulate(true, this, this.world, this.rand, chunkX, chunkZ, false);
```

Comment:
You should import

Reply:
Just following the way that it's done in the vanilla code. :)